packages/entities/entities-plugins/src/components/free-form/Datakit/flow-editor/node/FlowNode.vue

```diff
@@ -28,46 +28,55 @@
       :class="{ reversed: isReversed }"
     >
       <div class="input-handles">
-        <template v-if="data.fields?.input && data.fields.input.length > 0">
+        <template v-if="meta.io?.input">
           <div class="handle">
             <Handle
-              id="inputs"
+              id="input"
               :position="inputPosition"
               type="target"
             />
 
             <div class="handle-label-wrapper">
               <div
-                class="handle-label with-icon"
-                @click.stop="inputHandlesExpanded = !inputHandlesExpanded"
+                class="handle-label"
+                :class="{
+                  'has-fields': data.fields.input.length > 0,
+                  'not-collapsible': inputsNotCollapsible
+                }"
               >
                 <div>inputs</div>
```

Comment:
i18n, same as below

Reply:
I think we shouldn’t translate inputs here as it actually reflects a field (not input field) in the config.